sdk/nodejs/src/request.ts

```diff
@@ -47,6 +54,56 @@ export async function getEnvironmentConfig(
     })
 }
 
+export async function getAllFeatures(user: DVCPopulatedUser, envKey: string): Promise<AxiosResponse> {
+    return await post({
+        url: `${BUCKETING_URL}${HOST}${FEATURES_PATH}`,
+        headers: { 'Authorization': envKey },
+        data: user
+    })
+}
+
+export async function getAllVariables(user: DVCPopulatedUser, envKey: string): Promise<AxiosResponse> {
+    return await post({
+        url: `${BUCKETING_URL}${HOST}${VARIABLES_PATH}`,
+        headers: { 'Authorization': envKey },
+        data: user
+    })
+}
+
+export async function getVariable(user: DVCPopulatedUser, envKey: string, variableKey: string): Promise<AxiosResponse> {
+    return await post({
+        url: `${BUCKETING_URL}${HOST}${VARIABLES_PATH}/${variableKey}`,
+        headers: { 'Authorization': envKey },
+        data: user
+    })
+}
+
+    export async function postTrack(
+        user: DVCPopulatedUser,
+        event: DVCEvent,
+        envKey: string,
+        logger: DVCLogger
+    ): Promise<void> {
+    try {
+        const res = await post({
```

Comment:
this is only for tracking Custom Events, since the Variable Evaluated / Variable Defaulted events are tracked on the Bucketing API itself.. I don't believe it makes sense to leverage the existing event queue in this situation

Reply:
how come?

Reply:
we typically do not expect consecutive requests to the server to be from the same user, so we'd have many small event queues that would be flushed frequently... especially since this `track` method is ONLY for `customEvents`

Reply:
there's only one event queue that tracks all the events for all the users, and submits them in a batch

Reply:
ah.. then we probably can switch to using it

Reply:
yeah like it was designed to be an event queue for a serverside sdk, it collects all the events from the various track calls and then submits them, holds on to the ones that failed to submit and keeps trying. Eventually we could also consider having some kind of backing storage for the events